app/upgrades/v11/constants.go

```diff
@@ -1,20 +1,16 @@
 package v11
 
 import (
-	"github.com/osmosis-labs/osmosis/v10/app/upgrades"
-	twaptypes "github.com/osmosis-labs/osmosis/v10/x/gamm/twap/types"
-
 	store "github.com/cosmos/cosmos-sdk/store/types"
+
+	"github.com/osmosis-labs/osmosis/v10/app/upgrades"
 )
 
-// UpgradeName defines the on-chain upgrade name for the Osmosis v9 upgrade.
+// UpgradeName defines the on-chain upgrade name for the Osmosis v11 upgrade.
 const UpgradeName = "v11"
 
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
-	StoreUpgrades: store.StoreUpgrades{
-		Added:   []string{twaptypes.StoreKey},
-		Deleted: []string{}, // double check bech32ibc
```

Comment:
From what I remember, removing this breaks migrations so I would still keep the `Delete...` line to be 1:1 with `v11.x`